server/lib/configuration.js

```diff
@@ -25,6 +25,13 @@ var conf = module.exports = convict({
     doc: 'Check if the resources are under the /dist directory',
     format: Boolean
   },
+  babel: {
+    enabled: {
+      default: true,
+      doc: 'Convert ES2015 JavaScript to ES5',
```

Comment:
@shane-tomlinson yea, maybe add a note to the `doc` here so it won't confuse contributors (or us :D)